README.md

```diff
@@ -14,3 +21,15 @@ No configuration is required.
 - *is_valid_ipv4*: Check if an valid IPv4 address (with options to exclude loopback).
 - *is_valid_ipv6*: Check if an valid IPv6 address (with options to exclude loopback).
 - *is_valid_ip_port*: Check if an valid IP port (i.e. between 0 and 65535).
+- *geoip*: Report geoip infomation from MaxMind database(s).
+
+## GeoIP
+
+To use the `geoip` action you need to download one at least one of the
+following:
+
+- The (City geolite2)[http://dev.maxmind.com/geoip/geoip2/geolite2/] (licensed under CC BY-SA 4.0) database.
+- The paid (City)[https://www.maxmind.com/en/geoip2-city]database.
+- The paid (ISP)[https://www.maxmind.com/en/geoip2-isp-database] database.
+
```

Comment:
It would be good to document where this files need to be placed (aka the value set in the config).

Or does the `geop` library already do some (nasty) magic and places files into /opt/ when installing the PyPi package?